src/index.css

```diff
@@ -1,13 +1,9 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+@import url('https://fonts.googleapis.com/css2?family=Secular+One&display=swap');
```

Comment:
- [ ] The mobile view needs a little more styling:

![image](https://user-images.githubusercontent.com/57421823/227434105-6e292834-b181-496f-bb1d-8c8053e22d29.png)

I'm pretty sure this is a work in progress, and it's going to be awesome. 💪